zh/configure-a-tidb-cluster.md

```diff
@@ -116,7 +116,7 @@ TiDB Operator 支持为 PD、TiDB、TiKV 挂载多块 PV，可以用于不同用
     config:
       log:
         file:
-          name: /var/log/tidblog
+          filename: /var/log/tidblog
```

Comment:
cannot use a directory as the log file, how about using "/var/log/tidblog/tidb.log"

Reply:
done